src/main.zig

```diff
@@ -61,7 +61,8 @@ fn Generator(comptime ZigWriter: type, comptime JSWriter: type) type {
 
         fn generateJsScaffold(gen: *@This()) !void {
             try gen.js.writeAll(
-                \\const wasmMemoryBuf = undefined;
+                \\let wasm = undefined;
+                \\let wasmMemoryBuf = undefined;
```

Comment:
I think it'd be good to stick to Zig naming conventions even in our JS code perhaps; e.g. `wasm_memory_buf`?

Reply:
Agreed